collection_mutation.hh

```diff
@@ -113,3 +114,6 @@ public:
 collection_mutation merge(const abstract_type&, collection_mutation_view, collection_mutation_view);
 
 collection_mutation difference(const abstract_type&, collection_mutation_view, collection_mutation_view);
+
+// Serializes the given collection of cells to a sequence of bytes ready to be sent over the CQL protocol.
+bytes serialize_for_cql(const abstract_type&, collection_mutation_view, cql_serialization_format);
```

Comment:
I like the fact that it's a free function, it can later be moved to the native transport since that's the only thing it affects.